test/filewatch_test.erl

```diff
@@ -79,7 +93,7 @@ collect_events_loop(Files) ->
             NewFiles = ordsets:subtract(Files, [File]),
             collect_events_loop(NewFiles)
     after
-        10000 -> false
+        4500 -> false
```

Comment:
Minor nitpick but I feel like the return value could be improved a bit since it will directly influence the expected value in the tests. As an example, using a more descriptive atoms will make the test cases themselves more readable. Another, possibility is to return the list of files that were notified such that in the case of a failure, you have more information available for debugging.